studio/artifacts/artifact.py

```diff
@@ -9,13 +9,14 @@
     from urllib import urlopen
 from typing import Dict
 
```

Comment:
For all the massive amount of changes, they are mostly just fixes to modules import due to new packages structure.
There should not be any real change to functionality, at least none was intended.